src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -5958,5 +5958,21 @@ class C
                 }
                 """);
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70135")]
+        public void TestNormalizeLiteralCharacter()
+        {
+            var syntaxNode1 = SyntaxFactory.ParseExpression("1 is var i").NormalizeWhitespace();
+            Assert.Equal("1 is var i", syntaxNode1.ToFullString());
+
+            var syntaxNode2 = SyntaxFactory.ParseStatement("if (\"\" is var I)").NormalizeWhitespace();
+            Assert.Equal("if (\"\" is var I)", syntaxNode2.ToFullString());
+
+            var syntaxNode3 = SyntaxFactory.ParseStatement("if ('' is var I)").NormalizeWhitespace();
+            Assert.Equal("if ('' is var I)", syntaxNode3.ToFullString());
+
+            var syntaxNode4 = SyntaxFactory.ParseStatement("if ('x' is var I)").NormalizeWhitespace();
+            Assert.Equal("if ('x' is var I)", syntaxNode4.ToFullString());
```

Comment:
Would like also to see the following test cases:
- `@"" is var s`
- `"""a""" is var s`
- `$@"" is var s`
- `$"""a""" is var s`
- `""u8 is var s`

Reply:
i found some of them not working properly and i fix them